app/controllers/album.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const logger = require('../../app/logger');
+const PREVIOUS_MESSAGE = 'Making request..';
+const MESSAGE_OK = 'Request made correctly, sending ...';
```

Comment:
This messages seems like generic .. In this case, if tomorrow there is another controller and it needs to log this, you must to create in the new controller. Solution : Create a file `constants.js` and define all the constants generic.